backend/src/packages/chaiNNer_pytorch/pytorch/utility/convert_to_ncnn.py

```diff
@@ -43,7 +45,7 @@
     ],
 )
 def convert_to_ncnn_node(
-    model: PyTorchSRModel, is_fp16: int
+    model: SRModelDescriptor | RestorationModelDescriptor, is_fp16: int
```

Comment:
The `isinstance` checks below do not work anymore.

Also, since now have an arch name for each model, we could do this:
```py
assert not isinstance(model.model, (HAT, DAT, OmniSR, ...)), f"{model.architecture} is not supported for NCNN conversions at this time."
```